### PR TITLE
Add grey_elementAtIndex() [the internal n_th matcher] and disable a flaky alert view test.

### LINKED ISSUE #138: Add grey_atIndex(...) element matcher
Useful for correctly resolving duplicate element issues by index (picking first, last, etc.).

Background: https://github.com/google/EarlGrey/pull/135

`grey_atIndex(...)`

Use cases

- Find nth element in a table
- Resolve duplicate match

### PR DESCRIPTION
**Pull request by tirodkar**
Fix #138

### CODE REVIEW

**Comment by khandpur** [comment]
@bootstraponline PTAL

**Comment by bootstraponline** [comment]
Looks awesome!! Remember to update `cheatsheet.html`

**Comment by bootstraponline** [comment]
Also in the pull request description, if you write `Fix #138` then the issue will be auto-closed upon merge.

File: EarlGrey/Matcher/GREYMatchers.h [hunk 1]
```diff
@@ -488,4 +508,7 @@ GREY_EXPORT id<GREYMatcher> grey_lessThan(id value);
 /** Shorthand for GREYMatchers::matcherForGreaterThan:value. */
 GREY_EXPORT id<GREYMatcher> grey_greaterThan(id value);
 
+/** Shorthand for GREYMatchers::elementAtIndex:index. */
+GREY_EXPORT id<GREYMatcher> grey_elementAtIndex(NSUInteger index);
```
Comment (khandpur) [inline EarlGrey/Matcher/GREYMatchers.h]: GREYMatchers::matcherForElementAtIndex:.

no need to specify the parameter name like this: matcherForGreaterThan:value 

Can you please fix that to matcherForGreaterThan: ? there are a few others in this file that should be fixed.

**Comment by khandpur** [comment]
LGTM

**Comment by tirodkar** [comment]
Updated.

**Comment by khandpur** [comment]
@bootstraponline PTAL

**Comment by bootstraponline** [comment]
Looks good. 👍

**Pull request merged**